src/payment/data/selectors.js

```diff
@@ -47,7 +47,8 @@ export const paymentSelector = createSelector(
   queryParametersSelector,
   (basket, configuration, queryParameters) => {
     const isCouponRedeemRedirect =
-      queryParameters && queryParameters.coupon_redeem_redirect == 1; // eslint-disable-line eqeqeq
+      !!queryParameters &&
```

Comment:
Turning the first part into a proper bool so it returns false if it's undefined/null.